---
title: "Final Report: Your Title Here"
author: "Don Francisco"
date: "December 5, 2018"
output:
    github_document: default
bibliography: references.bib
csl: bioinformatics.csl
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

Add about 2-3 pages here. Across the whole manuscript, you should cite at least 20 peer reviewed articles.

# Methods

## Study design

WHAT DOES THIS SECTION ENTAIL



## Sample origin and sequencing

### Study Subjects 

Working at Eastman Institute of Oral Health, University of Rochester, Yu et al. recruited 23 current smokers with the following specifics: median duration, 15 years; median intensity, 15 cigarettes per day) and 20 nonsmokers who have smoked less than 100 cigarettes in their lifetime. Yu et al. excluded subjects who were using antibiotics at the time, had a professional dental cleaning within the last three months, or diagnosed with periodontal disease, which is a serious gum infection, or cancer. To determine if each participant needed to be excluded, the participants were screened by periodontal screening and recording (PSR) index at the time of recruitment. After all subjects were recruited, they signed a consent form and filled out questionnaires in relation to the study [@yu2017effect]. 

### Biospecimen Collection 

Yu et al. obtained nine samples, including supra and subgingiva plaque, saliva, swabs from five soft tissue areas, and one nasal swab from both nostrils, from each participant. They used the procedure of the Human Micriobiome Project to collect the samples. 

### 16S rRNA Gene Sequence Analysis 

After Yu et al. extracted the DNA from each sample, they amplified the V3-V4 regions of the 16S rRNA gene and sequenced it on an Illumina MiSeq instrument using the 300 paired-end protocol. They obtained this instrument from the Institute of Genome Sciences, Genomic Resource Center, University of Maryland School of Medicine and submitted the sequence data to NCBI BioProject.

Yu et al. processed the sequence reads in order to remove low quality and short reads. The remaining reads were bunched into Operational Taxonomy Units (OTUs) at 97% identity, and OTUs with only one read were excluded. They then estimated alpha diversity as the number of OTUs, Shannon's Index, and phylogenetic diversity through averaging >20 tables (1000 reads/sample). After this, they measured taxonomic beta diversity as unweighted and weighted UniFrac distance based on the OTU table. In order to rule out batch effects, or technical sources of differences that have been added to samples during handling, they selected 19 random samples in order to example the difference within and between each batch. 

### Statistical Analysis

Yu et al. used the Wilcoxon rank-sum test to examine gastric microbiota alpha diversity and taxa-relative abundance differences between each group. They then used the Spearman correlation to examine the relationship between continuous variables. Yu et al. used Bonferroni correction to adjust for tests of multiple taxa. They also used Permutational multivariate analysis of variance in order to look at the assocation between unweighted and weighted UniFrac distance and smoking status and other demographic variables. They considered P values of less than 0.05 to be significant after the adjustment for multiple tests. 

## Computational

These are the methods you used to do your bioinformatic analyses analyses. Should probably be between 0.5 and 1 pages. At a very minimum should include citations for DADA2 and phyloseq if you are doing an amplicon study, or other citations as appropriate.

# Results

## Subsections are ok in the results section too

```{r bar-graph-1, echo = FALSE}
melted_obj %>%
  group_by(Phylum, smoker) %>%
  summarize(sum_count = sum(Abundance)) %>%
  ggplot(aes(x = Phylum,
             y = sum_count,
             fill = smoker)) +
  geom_col(position = "dodge") +
  ggtitle("Hello") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
```

```{r bar-graph-2, echo = FALSE}
melted_obj %>%
  group_by(env_material, Phylum, smoker) %>%
  summarize(sum_count = sum(Abundance)) %>%
  filter(smoker == "smoker") %>%
  ggplot(aes(x = Phylum,
             y = sum_count,
             fill = env_material)) +
  geom_col(position = "dodge") +
  ggtitle("Hello") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
```

```{r bar-graph-3, echo = FALSE}
melted_obj %>%
  group_by(env_material, Phylum, smoker) %>%
  summarize(sum_count = sum(Abundance)) %>%
  filter(smoker == "smoker") %>%
  filter(Phylum == "Firmicutes") %>%
  ggplot(aes(x = Phylum,
             y = sum_count,
             fill = env_material)) +
  geom_col(position = "dodge") +
  ggtitle("Hello") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
```

```{r bar-graph-4, echo = FALSE}
melted_obj %>%
  group_by(env_material, Phylum, smoker, Family) %>%
  summarize(sum_count = sum(Abundance)) %>%
  filter(smoker == "smoker") %>%
  filter(Phylum == "Firmicutes") %>%
  ggplot(aes(x = Family,
             y = sum_count,
             fill = env_material)) +
  geom_col(position = "dodge") +
  ggtitle("Hello") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
```

```{r bar-graph-5, echo = FALSE}
melted_obj %>%
  group_by(env_material, Phylum, smoker, Family) %>%
  summarize(sum_count = sum(Abundance)) %>%
  filter(smoker == "smoker") %>%
  filter(Family == "Streptococcaceae") %>%
  ggplot(aes(x = Family,
             y = sum_count,
             fill = env_material)) +
  geom_col(position = "dodge") +
  ggtitle("Hello") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
```

```{r bar-graph-6, echo = FALSE}
melted_obj %>%
  group_by(env_material, Phylum, smoker, Family, host_subject_id) %>%
  summarize(sum_count = sum(Abundance)) %>%
  filter(smoker == "smoker") %>%
  filter(Family == "Streptococcaceae") %>%
  ggplot(aes(x = host_subject_id,
             y = sum_count,
             fill = env_material)) +
  geom_col(position = "dodge") +
  ggtitle("Hello") +
  theme(axis.text.x = element_text(angle = 90, hjust = 1))
```


```{r load-packages-and-data, echo = FALSE}
# load packages as needed
library("dplyr")
library("tidyr")
library("ggplot2")
library("knitr")
library("citr")

# load data produced from analysis scripts using
# something like load("output/processed_data.Rdata")
load("output/phyloseq_obj.Rdata")
load("output/melted_obj.Rdata")
```

In addition to a minimum of 5-8 figures/tables (and associated captions), you should include sufficient text in this section to describe what your findings were. Remember that in the results section you just describe what you found, but you don't interpret it - that happens in the discussion. 2-3 pages.

# Discussion

Add around 3-4 pages interpreting your results and considering future directions one might take in analyzing these data.

# Sources Cited
